schedule/yast/yast2_gui/yast2_gui_sle.yaml

```diff
@@ -52,6 +53,7 @@ conditional_schedule:
         - yast2_gui/yast2_lan_restart_vlan
         - yast2_gui/yast2_lan_restart_bond
       ppc64le:
+        - console/scc_cleanup_reregister
```

Comment:
what is the purpose of this module, only cleanup and reregister the default modules for its scenario or should be able as well to accept by test data more modules needed. just wondering if it could also do the work of installing development module in setup_libyui_running_system?

Reply:
We can, but I don't think it's a good idea because we possibly want to remove the dev module after the libyui setup and if we delegate the adding of the product to the general purpose module, it will be hard to distinguish weather we need to remove it or not. I will change the module a bit to accept test data, but for this test suite, no test data modification is needed. WDYT?

Reply:
yes, that makes sense, the development module could have been unregistered to not interfere with the remaining testing. So maybe you can even postpone adding test data because I don't know the case where we would like to add some specific set of modules, you can do it here or create a ticket for it. Thanks for the clarification.
Other than that, LGTM

Reply:
it's easy to add that functionality, even if it's not needed for now. I just have an issue with my workstation. I will run VRs with refspec